eg/Classes/PluginBase.py

```diff
@@ -148,7 +148,15 @@ def AddActionsFromList(self, theList, defaultAction=None):
         self.info.actionGroup.AddActionsFromList(theList, defaultAction)
 
     def AddEvents(self, *eventList):
-        self.info.eventList = eventList
+        if self.info.eventList is None:
+            self.info.eventList = ()
+
+        for event in eventList:
+            if isinstance(event, basestring):
+                event = (event, '')
+            elif isinstance(event, list):
+                event = tuple(event)
+            self.info.eventList += (event,)
```

Comment:
yes. the System plugin is one example.. if you wanted to make more then one call to AddEvents because you wanted to keep events in different files for the purpose of code organization. you couldn't before... because it would just overwrite it.. I also added a little bit of lazy to it as well.. so if the plugin dev just wants to pass a list of events because they do not want to add descriptions this will reformat it to work properly. idk it was simple to do.

but here is another example... as events occur. if it is not in the info.eventList it can be added dynamically with a simple method call.  instead of having to code this out. Like when you scan the tree in the AddEventsDialog.. that way you have a mechanism to add them to the list easily. but also as i mentioned above.